GithubJobApp/Job.swift

```diff
@@ -15,11 +15,11 @@ struct Job {
     let companyUrl: String?
     let description: String
     
-    init(title: String, company: String, description: String, companyLogoUrl: String? = nil, companyUrl: String? = nil) {
-        self.company = company
-        self.title = title
-        self.companyLogoUrl = companyLogoUrl
-        self.description = description
-        self.companyUrl = companyUrl
+    init(jsonJobDictionary: [String: AnyObject]) {
```

Comment:
Naming convention wise, it might be more common to see this first argument named `withDictionary:`  You also might add an internal name to be used within the init method, so the declaration would look like:

`init(withDictionary aDict: [String: AnyObject]) {`

And the the call site would look like 

`let foo = Job(withDictionary: jsonDict)`

and within the method, you would use `aDict` for the incoming dictionary.